packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -362,6 +365,9 @@ class DatePickerDialog extends StatefulWidget {
   /// Function to provide full control over which [DateTime] can be selected.
   final SelectableDayPredicate? selectableDayPredicate;
 
+  /// Describes what format will be used to display the date in the input box.
+  final intl.DateFormat? dateFormat;
```

Comment:
I think we should mention what will be the default date format if this is not provided.

Reply:
Agreed, I have added this.